networkconfig/config.go

```diff
@@ -30,15 +30,16 @@ func GetNetworkConfigByName(name string) (NetworkConfig, error) {
 }
 
 type NetworkConfig struct {
-	Name                    string
-	Beacon                  beacon.BeaconNetwork
-	Domain                  spectypes.DomainType
-	GenesisEpoch            spec.Epoch
-	RegistrySyncOffset      *big.Int
-	RegistryContractAddr    string // TODO: ethcommon.Address
-	Bootnodes               []string
-	WhitelistedOperatorKeys []string
-	RSAForkEpoch            spec.Epoch
+	Name                       string
+	Beacon                     beacon.BeaconNetwork
+	Domain                     spectypes.DomainType
+	GenesisEpoch               spec.Epoch
+	RegistrySyncOffset         *big.Int
+	RegistryContractAddr       string // TODO: ethcommon.Address
+	Bootnodes                  []string
+	WhitelistedOperatorKeys    []string
+	RSAForkEpoch               spec.Epoch
+	PermissionedDeactivateFork spec.Epoch
```

Comment:
wdyt about renaming this to `PermissionlessForkEpoch`? just to keep consistent naming for our fork epochs